micropip/utils.py

```diff
@@ -0,0 +1,49 @@
+from importlib.metadata import Distribution
+from pathlib import Path
+
+
+def get_dist_info(dist: Distribution) -> Path:
```

Comment:
Can you either make these functions private (starting with _) or make the whole module private?